core/protocol.py

```diff
@@ -62,3 +63,28 @@ def is_registered():
     """
     address = get_backend_address()
     return address == current_app.config['PUBLIC_KEY']
+
+def get_delivery(delivery_hash):
+    """
+    Returns owner, bytes_requested, and bytes_delivered for a delivery
+    """
+    d = get_datatrust()
+    delivery = call(d.get_delivery(delivery_hash))
```

Comment:
i think this can literally be `return call(d.get_delivery(delivery_hash))`

we can use destructuring on the other end of this call. i must have not conveyed my meaning very well...